library/model/borrowing.py

```diff
@@ -0,0 +1,54 @@
+from library.model.book import Book
+from library.model.user import User
+from datetime import datetime, timedelta
+from library.persistence.storage import LibraryRepository
+
+
+class Borrowing:
+    book: Book
+    user: User
```

Comment:
Is the user really necessary here? A Borrowing has a user and users have borrowings?